.github/workflows/test.yml

```diff
@@ -24,7 +24,7 @@ on:
         required: true
         type: string
       circt:
-        description: 'The CIRCT version to use (must be a valid release tag or "nightly")'
+        description: 'The CIRCT version to use (must be a valid release tag, "default", "nightly", or "version-file")'
```

Comment:
Default may be a bit ambiguous. This is probably fine as-is. Consider either making this empty/not a required option or something more descriptive like "use-firtool-resolver".

Reply:
Yeah good point. I think I'll go with not-required (thus empty) because that makes more sense that you're not installing a specific input, you're just leaving it up to chisel.